### ISSUE TITLE
TWIML Dial Typescript issues

### ISSUE THREAD

**Issue by MrBlenny** [twilio/twilio-node#384]
**Version:** 3.19.2

There seems to be an few issue with the ts twiml type for Dial:

```ts
  /**
   * <Dial> TwiML Verb
   *
   * @param attributes - TwiML attributes
   * @param number - Phone number to dial
   */
  dial(attributes: VoiceResponse.DialAttributes, number: string): VoiceResponse.Dial;
```

The [docs](https://www.twilio.com/docs/voice/twiml/number) suggest both of these are optional arguments yet the ts interface suggests they are required.

For example:
```ts
  const response = new twiml.VoiceResponse()
  const dial = response.dial({
    record: 'do-not-record'
  }, 'someNumber')
  dial.number({}, 'someNumber')
```

Results in:

```
<?xml version="1.0" encoding="UTF-8"?>
<Response>
    <Dial record="do-not-record">someNumber
        <Number>someNumber</Number>
    </Dial>
</Response>
````

Note the extra 'someNumber' introduced by the required arg in `response.dial`

---

Fix for now seems to be to pass in an empty string as the second param in `response.dial`

**Comment by daviddelucca** [comment]
@MrBlenny  Maybe I can give a hand in this issue tomorrow. I'll let you know.

**Comment by dkundel** [comment]
Thanks for catching this @MrBlenny! This code change has to be done in our Yoyodyne system since the file is auto-generated. I'll let the team know.

/cc @cjcodes

**Comment by childish-sambino** [comment]
Resolved by #419
